### ISSUE TITLE
drag & drop plugins not working in Opera 32.0, Chromium 46.0

### ISSUE THREAD

**Issue by lsh-0** [Strider-CD/strider#883]
Arch
Strider 1.7.5
Opera 32.0

I'm evaluating Strider as an alternate to our Jenkins platform and was a bit stymied how to get plain shell commands to run until I realized something with the drag and drop feature is broken in Opera in Chromium. It does work in Firefox 42.0

uBlock on Opera is installed, but still doesn't work when disabled. No errors in the console.

Perhaps fall back to a simpler mechanism for enabling plugins?

Let me know if I can help any further with this.

**Comment by knownasilya** [comment]
We don't support Opera, so all help is appreciated.

**Comment by lsh-0** [comment]
What do you support?

**Comment by knownasilya** [comment]
Yes, all major evergreen browsers. Chrome (although Chromium isn't exactly the same) is probably the best supported.

**Comment by lsh-0** [comment]
Ok. Opera has switched to Blink. Opera, Chrome and Chromium are pretty interchangeable these days and all fit the definition of 'evergreen'.

**Comment by knownasilya** [comment]
Feel free to submit PRs for issues with Opera, we will take them. This is a community project, so we cannot tackle all browsers if we have limited contributors. If you contribute regularly, then we will add you as a contributor to the project.

We would love for Strider to work in all browser :+1:

**Comment by oliversalzburg** [comment]
Apparently, they're not as interchangeable as you think ;) Funny thing actually, I just opened my Opera to see if I can reproduce the issue and the last open page is our Strider server :D

That being said, the drag & drop enable/disable for plugins works like a charm on 33.0.1990.58 for me.

**Comment by lsh-0** [comment]
grr. installing Opera 33 now. The answer may lie in the changelog:
http://blogs.opera.com/desktop/changelog-33/

...

nope, dragging but still not dropping.

On Fri, Nov 6, 2015 at 4:34 PM, Oliver Salzburg <notifications@github.com>
wrote:

> Apparently, they're not as interchangeable as you think ;) Funny thing
> actually, I just opened my Opera to see if I can reproduce the issue and
> the last open page is our Strider server :D
>
> That being said, the drag & drop enable/disable for plugins works like a
> charm on 33.0.1990.58 for me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Strider-CD/strider/issues/883#issuecomment-154461877>.
>



-- 

Luke Skibinski

A human being.
Slinging code, wrangling data.
Just being human.

--

eLife Sciences Publications, Ltd is a limited liability non-profit
non-stock corporation incorporated in the State of Delaware, USA, with
company number 5030732, and is registered in the UK with company number
FC030576 and branch number BR015634 at the address First Floor, 24 Hills
Road, Cambridge CB2 1JP.
